### DISCUSSION TITLE
Maybe something useful ?

### DISCUSSION THREAD

**Discussion by Jehova-Godfather** [flybywiresim/aircraft#2705] [Ideas]
Hi @all,
at first i want to say a big THANK YOU for this great mod. All of you did a very good job so far and it becomes better every day.

As i was a P3D Pilot in the past and got the FSLabs Airbus, let me say you were not far behind that 👍 

But there is one feature that i miss for my FMC.  Is it possible to cast the FMC via http? I used 2 old iPads to control the FMC and makes things much easier. :)

Best regarts
Markus

**Comment by andreda** [comment]
THAT would be amazing :)
